src/main/kotlin/net/leanix/vsm/sbomBooster/domain/BitBucketRepositoriesResponse.kt

```diff
@@ -25,6 +25,8 @@ data class BitBucketRepository(
     val name: String,
     @JsonProperty("links")
     val links: BitBucketRepositoryLinks,
+    @JsonProperty("uuid")
+    val uuid: String
```

Comment:
Is this `uuid` or `id`? I see in the GraphQL queries that we added an `id` property. Also, do we expect a String or a UUID? If the case is latter, maybe it's preferable to use the `UUID` class instead of `String`.

Reply:
@geoandri Bitbucket calls it a UUID, but their docs list the data type as a string. See [here](https://developer.atlassian.com/cloud/bitbucket/rest/api-group-repositories/#api-repositories-workspace-get):

> uuid
> string
> The repository's immutable id. This can be used as a substitute for the slug segment in URLs. Doing this guarantees your URLs will survive renaming of the repository by its owner, or even transfer of the repository to a different user.

Also, this code has been run and tested successfully while using this field as a string.

Reply:
Thanks for the info @dsimpson-leanix .